modules/roxctl-central-generate-k8s.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * command-reference/roxctl-central.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="roxctl-central-generate-k8s_{context}"]
+= roxctl central generate k8s
+
+Generate the required YAML configuration files to deploy Central into a Kubernetes cluster.
+
+.Usage
+[source,terminal]
+----
+$ roxctl central generate k8s [flags]
+----
+
+.Options
+[cols="6,3",options="header"]
+|===
+|Option |Description
+
+|`--central-db-image string`
+|Specify the Central database image you want to use. If not specified, a default value corresponding to the `--image-defaults` is used.
+
+|`--declarative-config-config-maps strings`
+|Specify a list of configuration maps that you want to add as declarative configuration mounts in Central.
+
+|`--declarative-config-secrets strings`
+|Specify a list of secrets that you want to add as declarative configuration mounts in Central.
+
+|`--enable-telemetry`
+|Specify whether you want to enable telemetry. The default value is `false`.
+
+|`--image-defaults string`
+|Specify the default settings for container images. It controls the repositories from which the images are downloaded, the image names and the format of the tags. The default value is `development_build`.
```

Comment:
```suggestion
|Specify the default settings for container images. The specified settings control the repositories from which the images are downloaded, the image names and the format of the tags. The default value is `development_build`.
```

Reply:
"the default settings" can't use "it" so I suggest this instead? Or something similar.